src/octokit/get-probot-octokit-with-defaults.ts

```diff
@@ -1,11 +1,12 @@
-import LRUCache from "lru-cache";
+import { type LRUCache } from "lru-cache";
 import { ProbotOctokit } from "./probot-octokit";
 import Redis from "ioredis";
 
 import { getOctokitThrottleOptions } from "./get-octokit-throttle-options";
 import { aliasLog } from "../helpers/alias-log";
```

Comment:
Is aliasLog still needed in this file? Otherwise, it could be dropped entirely.

Reply:
It's not needed. It can be removed